pkg/roachpb/io-formats.proto

```diff
@@ -32,8 +32,17 @@ message IOFileFormat {
   optional CSVOptions csv = 2 [(gogoproto.nullable) = false];
   optional MySQLOutfileOptions mysql_out = 3 [(gogoproto.nullable) = false];
   optional PgCopyOptions pg_copy = 4 [(gogoproto.nullable) = false];
+
+  enum Compression {
+    Auto = 0;
+    None = 1;
+    Gzip = 2;
+    Bzip = 3;
```

Comment:
Zip is in the standard library. Can we add that too? Or is it annoying because it could contain multiple files?